doc/docs/master/concepts/pipeline-concepts/job.md

```diff
@@ -1,16 +1,128 @@
+job runs your code against the current commit in a `<repo>@<branch>` and
+then submits the results to the output repository of the pipeline as a single output commit. A pipeline
 triggers a new job every time you submit new changes, a commit, into your
 input source.
 
+Each job has an alphanumeric identifier (ID) that you can reference in the `<pipeline>@<jobID>` format.
+
+You can obtain information about all jobs with a given ID by running `list job <jobID>` or restrict to a particular pipeline `list job -p <pipeline>`, or `inspect job <pipeline>@<jobID> --raw`.
+
+## Job Statuses
 Each job has the following stages:
 
 | Stage     | Description  |
 | --------- | ------------ |
 | Starting  | Pachyderm starts the job when it detects new data in the input repository. <br> The new data appears as a commit in the input repository, and Pachyderm <br> automatically launches the job. Pachyderm spins the number of Pachyderm worker pods <br> specified in the pipeline spec and spreads the workload among them. |
 | Running   | Pachyderm runs the transformation code that is specified <br> in the pipeline specification against the data in the input commit. |
 | Merging   | Pachyderm concatenates the results of the processed <br> data into one or more files, uploads them to the output repository, completes the final output commits, and creates/persists all the versioning metadata |
+
+## List Jobs
+
+- The `pachctl list job` command returns list of all global jobs. This command is detailed in [this section of Global ID](../globalID/#list-all-global-commits-and-global-jobs).
+
+- The `pachctl list job <jobID>` commands returns the list of all jobs sharing the same `<jobID>`. This command is detailed in [this section of Global ID](../globalID/#list-all-commits-and-jobs-with-a-global-id). 
+
+- Note that you can also track your jobs downstream as they complete by running `pachctl wait job <jobID>`. 
+
+- The `pachctl list job -p <pipeline>` command returns the jobs in a given pipeline.
+
+    !!! example
+        ```shell
+        $ pachctl list job -p edges
+        ```
+
+        **System Response:**
+
+        ```shell
+        ID                               PIPELINE STARTED      DURATION           RESTART PROGRESS  DL       UL       STATE
+        fd9454d06d8e4fa38a75c8cd20b39538 edges    20 hours ago 5 seconds          0       2 + 1 / 3 181.1KiB 111.4KiB success
+        5a78358d4b53494cbba4550428f2fe98 edges    20 hours ago 2 seconds          0       1 + 0 / 1 57.27KiB 22.22KiB success
+        7dcd77a2f7f34ff384a6096d1139e922 edges    20 hours ago Less than a second 0       0 + 0 / 0 0B       0B       success
+        ```
+
+    For each job, Pachyderm shows the time the pipeline started with its duration, data downloaded and uploaded, STATE of the pipeline execution, the number of datums in the **PROGRESS** section,  and other information.
+    The format of the progress column is `DATUMS PROCESSED + DATUMS SKIPPED / TOTAL DATUMS`.
```

Comment:
unless it's elsewhere in the docs, could be good to note here that failed datums are only included in the "denominator", here, so it might look like the numbers don't add up

Reply:
Good point - It is in this page: `concepts/pipeline-concepts/datum/datum-processing-states.md`